packages/botonic-cli/tests/commands/deploy.test.ts

```diff
@@ -43,7 +43,19 @@ describe('TEST: Deploy pipeline', () => {
         const onceBundled = readDir('.')
         expect(onceBundled).toContain('botonic_bundle.zip')
         expect(onceBundled).toContain('tmp')
-        const { hasDeployErrors } = await deployCommand.deployBundle()
+        const botConfigJson = {
+          build_info: {
+            node_version: 'v20.0.0',
+            npm_version: '10.0.0',
+            botonic_cli_version: '0.29.0',
+          },
+          packages: {
+            '@botonic/core': '0.29.0',
+            '@botonic/react': '0.29.0',
+          },
+        }
```

Comment:
this is a default/base config to start from right? is this really needed?

Reply:
if so, it would be nice to have these versions in a separate file to be easier to maintain

Reply:
this is an example for the test